.github/workflows/main-suite.yml

```
@@ -72,7 +72,7 @@ jobs:
         run: yarn test:ci:coverage
 
       - name: Test Public Typings
-        run: yarn test:tsd
+        run: yarn test:types
```

As an example, you can say that TypeScript 5.4 and above is supported and run tests like this:

```suggestion
        run: yarn test:types --target '>=5.4'
```

The tests will run agains each minor release, currently: `5.4`, `5.5`, `5.6`, `5.7`, `5.8`. New TypeScript versions are added automatically as soon as they are published to npm.